lib/kernel/test/Makefile

```diff
@@ -217,7 +217,7 @@ release_tests_spec: make_emakefile
 	$(INSTALL_DATA) $(ERL_FILES) $(HRL_FILES) "$(RELSYSDIR)"
 	$(INSTALL_DATA) $(APP_FILES) $(EXTRA_FILES) "$(RELSYSDIR)"
 	$(INSTALL_DATA) \
-		kernel.spec kernel_smoke.spec kernel_bench.spec logger.spec \
+		kernel.spec kernel_smoke.spec kernel_bench.spec kernel_gh.spec logger.spec \
```

Comment:
It is a list of all files to be copied when releasing tests. There is an overlap between `kernel.spec` and `kernel_gh.spec` because it is not possible to include specs into each other (as far as I know).

`$APP.spec` is what is normally run, while in GH actions we run `$APP_gh.spec` if it exists.